guides/deployments/podman.md

```diff
@@ -127,79 +128,124 @@ Linux kernel doesn't support CVMs.
    kubectl apply -f ./smarter-device-manager.yaml
    ```
 
+   The example `DaemonSet` includes a `nodeSelector` that constrains the device
+   plugin to nodes with the `smarter-device-manager` label set to `enabled`.
+   Label the nodes that will include the FUSE device by using the following
+   command, or remove the `nodeSelector` from the manifest:
+
+   ```console
+   kubectl get nodes
```

Comment:
this was my fault and it's pretty minor but this code block gets a Copy button, but the resulting snippet isn't useful on its own since the node name needs to be replaced.

Looks like we can replace this with the following, to label all nodes accordingly: `kubectl label nodes --all smarter-device-plugin=enabled`

Reply:
@khorne3 my bad, I gave you bad directions in the above comment, the code I had was right but the comment was wrong

```
         nodeSelector:
           smarter-device-manager: enabled
```

This means the label should be `smarter-device-manager=enabled` instead of `smarter-device-plugin` :man_facepalming: